paddlers/utils/postprocs/connection.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+
+warnings.filterwarnings("ignore", category=DeprecationWarning)

```

Comment:
这个是为了滤除什么警告呢？

Reply:
应该是`sklearn`和`numpy`的版本问题，像这样

```
D:\ProgramData\Anaconda3\envs\paddlers\lib\site-packages\sklearn\xxxxx\xxx.py:xx: DeprecationWarning: `np.float` is a deprecated 
alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

Reply:
这样的话我不建议一刀切（因为paddlers自动import这个模块后，可能会把其它地方本来可以暴露出来的警告也给埋没了），可以针对特定的模块处理一下，[warnings标准库是支持这个功能的](https://docs.python.org/zh-cn/3/library/warnings.html#the-warnings-filter)。

Reply:
看是使用这个上下文管理器可以吗晖哥